doc/ref/chap-type-method.md

```diff
@@ -633,13 +633,69 @@ Ensures that the given key does not exist in the dictionary.
     = book
     # => (Dict)   {title: 'The Histories'}
 
-### accum()
+### append()
 
-TODO:
+Appends the given value to the list pointed to by the key:
 
-```raw
-call mydict->accum('key', 'string to append')
-```
+    var mydict = {a: [1]}
+    = mydict
+    # => (Dict)   {a: [1]}
```

Comment:
hm I usually write `# =>` on the right, but I guess having it on the next line can be clearer in some cases